services/workflows-service/scripts/run-filter.ts

```diff
@@ -0,0 +1,56 @@
+import { PrismaClient } from '@prisma/client';
+
+export async function fliterQuery() {
```

Comment:
What is this file? Should it be in the source code?

Reply:
It's a script/tool I used it to try fliters before creating them, should be part of the source code (similar to seed.ts)